ods_ci/tests/Tests/100__deploy/130__operators/140__serverless_operator/140__serverless_operator.robot

```diff
@@ -0,0 +1,116 @@
+*** Settings ***
+Documentation    Test Cases to verify Serverless installation
+Library         Collections
+Library         SeleniumLibrary
+Library         OpenShiftLibrary
+Resource        ../../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+Suite Setup      Suite Setup
+Suite Teardown   Suite Teardown
+
+
+*** Variables ***
+${SERVERLESS_APPNAME}  Red Hat OpenShift Serverless
+${SERVERLESS_OPERATOR_NAME}    Red Hat OpenShift Serverless
+${KNATIVESERVING_NS}    knative-serving
+${ISTIO_NS}     istio-system
+
+
+*** Test Cases ***
+Validate DSC creates all Serverless CRs
+    [Documentation]  The purpose of this Test Case is to validate the creation
+    ...    of Serverless Custom Resources
+    [Tags]  Operator    ODS-2600
+    Assign Vars According To Product    ${PRODUCT}
+    Check And Install Operator in Openshift    ${OPERATOR_APPNAME}    ${OPERATOR_NAME}
+    Check And Install Operator in Openshift    ${SERVERLESS_APPNAME}    ${SERVERLESS_OPERATOR_NAME}
```

Comment:
optional: Isn't better to first install serverless and then rhoai since serverless is supposed to be a dependency for rhoai?

Reply:
Yes @jstourac this is meant for Platform Team Validation

Reply:
I can switch order